Makefile.am

```diff
@@ -163,7 +164,18 @@ clean-storage:
 all-local: \
 	vdsm.spec
 
-.PHONY: srpm rpm
+.PHONY: config
+config:
+	./autogen.sh --system --enable-timestamp
```

Comment:
This does not work, since the Makefile is generated by running ./autogen.sh.

Reply:
True. It is a pity, it led to a nice solution where all the command were executed through dependencies. But if we cannot do it, we would need to skip the `clean config` targets.

I tested locally and seemed to work, but I didn't notice that for the next steps it was using the Makefile stored in memory and not the freshly generated.